### DISCUSSION TITLE
Super helpful, thanks! Just one suggestion for the instructions...

### DISCUSSION THREAD

**Discussion by davin900** [john-stephens/zebra-mac-label-automator#10] [Ideas]
I followed the instructions and they were super easy to understand/follow. The only thing is that my Mac did not have a PDF Services folder. So I just created one and it worked fine. But you might want to clarify that in the instructions. I wasn't sure if I was looking in the wrong place at first. I'm on an M1 MBP running the latest OS if that matters.

**Comment by john-stephens** [comment]
Instructions updated. Thanks for the feedback!
